### ISSUE TITLE
Makefile fails to detect compilers...that are present

### ISSUE THREAD

**Issue by TheFakeMontyOnTheRun** [pd95/m68k-atari-mint-cross-tools#1]
Hello!
This project seems very nice, but it fails to detect the working compilers:
```
checking whether /Users/monty/gnu-tools/m68000/bin/m68k-atari-mint-gcc is gcc... yes
checking compiler /Users/monty/gnu-tools/m68000/bin/m68k-atari-mint-gcc -m32 -O2 -pedantic -fomit-frame-pointer ... no
checking compiler /Users/monty/gnu-tools/m68000/bin/m68k-atari-mint-gcc -O2 -pedantic -fomit-frame-pointer ... no, program does not run
configure: error: could not find a working compiler, see config.log for details
make[1]: *** [/Users/monty/git/m68k-atari-mint-cross-tools/compile/gmp-6.2.0-bin-darwin-x86_64/Makefile] Error 1
make: *** [/Users/monty/git/m68k-atari-mint-cross-tools/archive/gcclibs/i386] Error 2
```
Meanwhile, I'm pretty capable of calling (with the exception of the parameter -m32 not being supported).
```
MontyMacBookAir% ~/gnu-tools/m68000/bin/m68k-atari-mint-gcc                                                                                                                                                               on branch master
m68k-atari-mint-gcc: fatal error: no input files
compilation terminated.
MontyMacBookAir% /Users/monty/gnu-tools/m68000/bin/m68k-atari-mint-gcc -m32 -O2 -pedantic -fomit-frame-pointer                                                                                                            on branch master
m68k-atari-mint-gcc: error: unrecognized command line option '-m32'
m68k-atari-mint-gcc: fatal error: no input files
compilation terminated.
MontyMacBookAir% /Users/monty/gnu-tools/m68000/bin/m68k-atari-mint-gcc -O2 -pedantic -fomit-frame-pointer                                                                                                                 on branch master
m68k-atari-mint-gcc: fatal error: no input files
compilation terminated.
```

This is MacOS Mojave 10.14.6

**Comment by pd95** [comment]
Hi Daniel

I'm not sure what you're trying to do. From your excerpt it seems you have already a m68k-atari-mint-gcc compiler in `/Users/monty/gnu-tools/m68000/bin/`. 

What are you trying to achieve by building gcclibs respectively gcc from `/Users/monty/git/m68k-atari-mint-cross-tools/`? The goal of these makefiles is to bootstrap a m68k-atari-mint cross-compiler environment.

Regards
Philipp

PS: I'm in the process of testing the scripts on macOS 11 with Xcode 12... they need some fixes as many packages have been updated in the source http://vincent.riviere.free.fr/soft/m68k-atari-mint/

**Comment by TheFakeMontyOnTheRun** [comment]
I was following the steps described in the README.md.
TBH, I already had a working compiler (but without libs) from https://github.com/mikrosk/m68k-atari-mint-build, but that should not have interfered with it

I've noticed some of the filenames have changed and had some adjustments made here was well. I actually hoped to submit a PR once everything was working, but seems like there is no need for it

**Comment by pd95** [comment]
Ah, OK. So you try to build the "additional libraries" like gemlib, cflib and qed and you would like to use the compiler which is located `~/gnu-tools/m68000/bin/`.

I really don't know if this will/could work, as the script is simply building all the tool even though you might have them already somewhere... there is no check for existing cross-compilers. 

But you can simply try `make gemlib`. This would fetch the gemlib library and build it. If you inspect the `Makefile.gemlib` you will see that it will basically run the following command

    make --directory="$(SOURCE_DIR)" CROSS=yes PREFIX=$(LOCAL_PREFIX_DIR)/m68k-atari-mint

It's the flag `CROSS=yes`, which is handled by the regular gemlib Makefile (from the source directory), that makes the build use the cross compiler.

So in short: If you have your cross-compiler already accessible in your environments PATH (e.g. typing `m68k-atari-mint-gcc --version` works), you can simply use the following commands to build the libraries and qed:

    make gemlib
    make cflib
    make qed

(Each Makefile target has also its own `Makefile.<target>` doing the actual work)

**Comment by TheFakeMontyOnTheRun** [comment]
Weirdly, after cleaning up everything and starting over, everthing worked perfectly!
Sorry about the confusion. Thanks!

PS: the only adjustment I needed was to change GMP tar.lz file to tar.bz2 on Makefile.gcclibs (I'm on Mojave)
